### PR TITLE
03 server operational

### PR DESCRIPTION
**Pull request by guillaume-boutin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,42 @@ app.get('/', function (req, res) {
   res.send('Hello World!');
 });
 
-
+app.get('/:operation/:num1/:num2', function(req, res) {
```
Comment (nyanofthemoon) [inline index.js]: "Create a web serverthat can listen to requests for /op/:operation/:number1/:number2"

File: index.js [hunk 2]
```diff
@@ -5,7 +5,42 @@ app.get('/', function (req, res) {
...
+  
+  var operator = req.params.operation;
+  var num1 = Number(req.params.num1);
+  var num2 = Number(req.params.num2);
+  var result;
+  
+  switch (operator) {
```
Comment (nyanofthemoon) [inline index.js]: What happens if my operator is "potato"? :)

**Comment by guillaume-boutin** [comment]
Mon server va faire patate

**Comment by ziad-saab** [comment]
@guillaume-boutin not 100% yet. What about the whole part of the exercise that talks about `res.sendStatus`?

File: index.js [hunk 3]
```diff
@@ -5,7 +5,45 @@ app.get('/', function (req, res) {
...
+      } else {
+        res.status(400).send('Error 400: I cannot divide by zero!');
+      }
+      
+    case 'mod':
+      result = num1%num2;
+      res.send(num1+' % '+num2+' = '+result);
+      
+    default:
+      res.status(400).send("Error 400: That is not an operation i know of.");
```
Comment (nyanofthemoon) [inline index.js]: :+1:

**Pull request closed without merging**